libafl/src/stages/calibrate.rs

```diff
@@ -60,11 +60,16 @@ impl UnstableEntriesMetadata {
     }
 }
 
+/// Default name for `CalibrationStage`
+pub const CALIBRATION_STAGE_NAME: &str = "CalibrationStage";
```

Comment:
As discussed, this guy can go

Reply:
@domenukk AFL variant as default just for calibration? or for sync & colorization too?

Reply:
you don't need this name in my opinion. just keep 
pub const CALIBRATION_STAGE_NAME: &str = "calibration"
and then it's fine.

Reply:
> In any case, I think we may want a constructor for custom names down the road else we can't have multiple instances of the same stage (right?)

we can have multiple same stages. but more imporantly we don't use name anymore because we have the handle. so it doesn't hatter

Reply:
@R9295 you make afl name default for all